Mathlib/Analysis/Calculus/ContDiff/Defs.lean

```diff
@@ -162,7 +162,7 @@ theorem contDiffWithinAt_nat {n : ℕ} :
 /-- When `n` is either a natural number or `ω`, one can characterize the property of being `C^n`
 as the existence of a neighborhood on which there is a Taylor series up to order `n`,
 requiring in addition that its terms are analytic in the `ω` case. -/
-lemma contDiffWithinAt_iff_of_ne_infty (hn : n ≠ ∞) :
+lemma contDiffWithinAt_iff_of_ne_infty (hn : n ≠ ∞ := by finiteness) :
```

Comment:
I don't think `finiteness` will ever work in these `ContDiff` files, because the smoothness index lives in `WithTop ℕ∞`, with possible values a natural, or ∞, or ω.